src/mono/mono/mini/llvm-intrinsics.h

```diff
@@ -394,10 +399,14 @@ INTRINS_OVR_TAG(AARCH64_ADV_SIMD_FRSQRTE, aarch64_neon_frsqrte, Arm64, Scalar |
 INTRINS_OVR_TAG(AARCH64_ADV_SIMD_FRSQRTS, aarch64_neon_frsqrts, Arm64, Scalar | V64 | V128 | R4 | R8)
 INTRINS_OVR_TAG(AARCH64_ADV_SIMD_FRECPS, aarch64_neon_frecps, Arm64, Scalar | V64 | V128 | R4 | R8)
 
+#if LLVM_API_VERSION < 1400
```

Comment:
Should we just remove this when switching to LLVM 14? Without the definition, this intrinsics won't work, right?

Reply:
Removed it, its not used anywhere.

Reply:
It is used here https://github.com/dotnet/runtime/blob/main/src/mono/mono/mini/simd-intrinsics.c#L2520

Reply:
I am also curious why it stops working for llvm14+?

Reply:
It got replaced by a general arch independent bitreverse intrinsic.